services/src/handlers/workflows.rs

```diff
@@ -120,34 +102,17 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn load() {
-        let ctx = InMemoryContext::default();
+    async fn register_invalid_method() {
+        let res = register_test_helper("GET").await;
```

Comment:
Currently you only test whether GET methods return 405 on POST and vice versa but there are more HTTP methods. Can you please try to find a way to check a handler against all HTTP methods? I am thinking of a function with a request and a list of allowed methods and it tests if all other HTTP methods return 405.

Reply:
f47640f
02f4ddf